src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -57,6 +64,23 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_DELETE_PATIENT_SUCCESS, Messages.format(personToDelete)));
     }
 
+    /**
+     * Returns the doctor that contains the given appointment.
```

Comment:
English in javadoc is incoherent? Maybe Doctor that has the appointment? minor nits